model_analyzer/record/types/gpu_free_memory.py

```diff
@@ -13,11 +13,11 @@
 # limitations under the License.
 
 from functools import total_ordering
-from model_analyzer.record.gpu_record import GPURecord
+from model_analyzer.record.gpu_record import GPURecord, IncreasingGPURecord
 
 
 @total_ordering
-class GPUFreeMemory(GPURecord):
```

Comment:
What was implemented in GPURecord? Are we losing that?

Reply:
We're not losing any functionality. I've just augmented GPURecord to have increasing and decreasing records.

Reply:
ok, I see now that it is IncreasingGPURecord 👍